pip/pip-375.md

```diff
@@ -0,0 +1,41 @@
+# PIP-375: Expose the Admin client configs: readTimeout, requestTimeout, connectionTimeout
+
+# Background knowledge
+
+- `pulsar-admin sinks create` will upload files to brokers, but if the file is large and the network is slow, it may cost a long time.
+- The default value of `requestTimeout` of Admin API is `60s`, after the fix https://github.com/apache/pulsar/pull/23128, it was changed to `300s`.
+
+# Motivation
+
+We'd better to expose the following configs to users:
+- `readTimeout`
+- `requestTimeout`
+- `connectionTimeout`
```

Comment:
Please add a short definition of these configuration parameters.

One reason is that there has been misconceptions about readTimeout and requestTimeout as described in #23128.

There's also a possibility for confusion about "connection timeout". One has to think whether it means "connect timeout" or "connection idle timeout", or something else. Based on the javadoc it means a "connect timeout".

Reply:
+1

Reply:
Added description